src/tracks/InstrumentTrack.cpp

```diff
@@ -124,6 +124,7 @@ InstrumentTrack::InstrumentTrack( TrackContainer* tc ) :
 	connect( &m_baseNoteModel, SIGNAL( dataChanged() ), this, SLOT( updateBaseNote() ) );
 	connect( &m_pitchModel, SIGNAL( dataChanged() ), this, SLOT( updatePitch() ) );
 	connect( &m_pitchRangeModel, SIGNAL( dataChanged() ), this, SLOT( updatePitchRange() ) );
+	connect( &m_mutedModel, SIGNAL( dataChanged() ), this, SLOT( muteHasChanged() ) );
```

Comment:
Or better yet, perhaps the mute color change slot should simply be created in the construction of the view class only?